fetch.bs

```diff
@@ -5746,9 +5764,6 @@ constructor must run these steps:
    <a for=request>use-CORS-preflight flag</a>.
   </ol>
 
- <li><p>If <var>inputBody</var> is <var>body</var> and <var>input</var> is <a for=Body>disturbed</a>
- or <a for=Body>locked</a>, then <a>throw</a> a {{TypeError}}.
```

Comment:
Don't we still need this? `input.body` is not itself extracted, only `init.body`, so the change would only affect `init.body`.

Reply:
Oh, I blindly trusted https://github.com/whatwg/fetch/issues/792#issuecomment-413898585. I will double check.

Reply:
Sorry about that! I should have double checked before writing that.